src/db_impl.cc

```diff
@@ -397,11 +397,26 @@ Status TitanDBImpl::DropColumnFamilies(
     column_families.emplace_back(handle->GetID());
     column_families_str += "[" + handle->GetName() + "]";
   }
+  {
+    MutexLock l(&mutex_);
+    drop_cf_requests_++;
+
+    // Has to wait till no GC job is running before proceed, otherwise GC jobs
+    // can fail and set background error.
+    // TODO(yiwu): only wait for GC jobs of CFs being dropped.
+    while (bg_gc_running_ > 0) {
+      bg_cv_.Wait();
```

Comment:
a column family is actually removed in `DestroyColumnFamilyHandle`, we should wait there instead

Reply:
We call `rocksdb::DropColumnFamilies()` of base db in `tiandb::DropColumnFamilies()`, after which GC will fail to write back to LSM, causing background error. We need to wait here.